DEPLOYNOTES.rst

```diff
@@ -4,6 +4,14 @@ Deploy and Upgrade notes
 ========================
 
 
+0.22
+----
+
+* Solr schema has been changed and should be updated::
```

Comment:
do we want to be specific about why? minor thing, but i feel like in the past the note has been "addition of X field requires a re-index" or something similar

Reply:
Good point. Sure, will add.